README.md

```diff
@@ -160,6 +160,23 @@ Note that you can
 
 #### Instructions
 
+##### Optionally building LLVM
```

Comment:
I'd like to omit this new section. The readme is a quick start guide. The supported method of discovering advanced build options is to read CMakeLists.txt. Teaching people how to install system dependencies is out of scope of the Zig readme. The supported method of installing Zig for people who do not know how to build dependencies from source or use their system package manager is to use the pre-built binaries from the download page.